packages/angular/src/flyout-menu/index.ts

```diff
@@ -0,0 +1,3 @@
+export { FlyoutMenu } from './flyout-menu.component';
+export { FlyoutMenuPane } from './flyout-menu-pane.component';
+export { FlyoutMenuModule } from './flyout-menu.module';
```

Comment:
Also make sure there's a `package.json` in the `flyout-menu` directory - that way it's exposed as a separate entrypoint ... Also, it should be exported from the root `index.ts`